http/vibe/http/server.d

```diff
@@ -532,7 +532,7 @@ alias HTTPServerErrorPageHandler = void delegate(HTTPServerRequest req, HTTPServ
 */
 enum HTTPServerOption {
```

Comment:
When this module is rewritten, it probably makes sense to rename / fully deprecate this enum as sadly AFAIK individual fields can't be deprecated at the moment

Reply:
Oh we can hack around it: https://github.com/vibe-d/vibe.d/pull/1947

Reply:
I really hope that this will be implemented one day, as this is far from being the only enum with deprecated fields in vibe.d.